app/src/androidTest/java/com/android/partagix/DatabaseTests.kt

```diff
@@ -390,6 +390,54 @@ class DatabaseTests {
     unmockkStatic(::now)
   }
 
+  @Test
+  fun testCreateLoan() {
+    mockkStatic(::now)
+    every { now() } returns Timestamp(Date(0))
+
+    val taskCompletionSource = TaskCompletionSource<Void>()
+
+    val mockCollection = mockk<CollectionReference>()
+
+    val mockDocument = mockk<DocumentReference>()
+
+    every { mockCollection.document(any()) } returns mockDocument
+
+    every { mockCollection.document() } returns mockDocument
+
+    val documentId = "wkUYnOmKkNVWlo1K8/59SDD/JtCWCf9MvnAgSYx9BbCN8ZbuNU+uSqPWVDuFnVRB"
+    every { mockDocument.id } returns documentId
+
+    every { mockDocument.set(any()) } returns
+        taskCompletionSource.task.continueWith(Executors.DIRECT_EXECUTOR, voidErrorTransformer())
+
+    val mockDb: FirebaseFirestore = mockk {}
+
+    every { mockDb.collection(any()) } returns mockCollection
+
+    val database = spyk(Database(mockDb), recordPrivateCalls = true)
+
+    val loan =
+        Loan(
+            "id",
+            "id_owner",
+            "id_loaner",
+            "id_item",
+            Date(0),
+            Date(0),
+            "r",
```

Comment:
les reviews sont des nombres il faudrait plutot mettre "0.0" pour la cohérence des tests

Reply:
It makes sense, it's changed.